### PR TITLE
Filter non-active service name when UI query.

### PR DESCRIPTION
**Pull request by peng-yongsheng**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [X] New feature provided
- [ ] Improve performance

- Related issues
#1124 
___
### New feature or improvement
- Describe the details and related test reports.

Add heart beat time into service name index to filter non-active service name.

### CODE REVIEW

**Comment by wu-sheng** [comment]
And 24 hours look like too unpredict to end user. I suggest we support ttl-day, ttl-year to remove data. And the service status of valid could be considered by the ttl-*.

**Comment by wu-sheng** [comment]
I think we can move on this. Based on #1268 TTL settings.

File: apm-collector/apm-collector-storage/collector-storage-es-provider/src/main/java/org/apache/skywalking/apm/collector/storage/es/dao/InstanceHeartBeatEsPersistenceDAO.java [hunk 1]
```diff
@@ -51,10 +51,17 @@ public InstanceHeartBeatEsPersistenceDAO(ElasticSearchClient client) {
...
             logger.debug("instance id: {} is exists", id);
             return instance;
         } else {
             logger.debug("instance id: {} is not exists", id);
+=======
+            logger.debug("instance id: {} exists", id);
+            return instance;
+        } else {
+            logger.debug("instance id: {} not exists", id);
+>>>>>>> master
```
Comment (wu-sheng) [inline apm-collector/apm-collector-storage/collector-storage-es-provider/src/main/java/org/apache/skywalking/apm/collector/storage/es/dao/InstanceHeartBeatEsPersistenceDAO.java]: @peng-yongsheng Look like conflicts.

**Comment by peng-yongsheng** [comment]
1. Add application id to be a query condition of searchService method.
2. Filter service name by given time which calculated by the minute metric TTL setting. ( CurrentTimeStamp - minuteMetricTTL )

**Comment by wu-sheng** [comment]
@peng-yongsheng  ci fails.

**Pull request merged**